bar-trender-front-end/src/views/ListPage.js

```diff
@@ -0,0 +1,229 @@
+import React from "react";
+
+// reactstrap components
+import {
+  Button,
+  NavItem,
+  NavLink,
+  Nav,
+  TabContent,
+  TabPane,
+  Container,
+  Row,
+  Col,
+  UncontrolledTooltip,
+} from "reactstrap";
+
+// core components
+import MainNavbar from "../components/Navbars/MainNavbar.js";
+import ProfilePageHeader from "../components/Headers/ProfilePageHeader.js";
+import DefaultFooter from "../components/Footers/DefaultFooter.js";
+
+import image_1 from '../assets/img/bg3.jpg';
+import image_2 from '../assets/img/bg4.jpg';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7123021)
```suggestion
import image_2 from "../assets/img/bg4.jpg";
```